compute/src/test/java/org/jclouds/compute/callables/InitScriptConfigurationForTasksTest.java

```diff
@@ -76,4 +73,19 @@ public void testIncrementingTimeSupplier() throws InterruptedException {
       assertEquals(config.getAnonymousTaskSuffixSupplier().get(), "1");
    }
 
+   @Test
+   public void testInitScriptPattern() throws Exception {
+      InitScriptConfigurationForTasks config = InitScriptConfigurationForTasks.create();
+      config.initScriptPattern("/var/tmp/jclouds-%s");
+      assertEquals(config.getBasedir(), "/var/tmp");
+      assertEquals(config.getInitScriptPattern(), "/var/tmp/jclouds-%s");
+   }
+
+   @Test
+   public void testInitScriptPatternAtRoot() throws Exception {
+      InitScriptConfigurationForTasks config = InitScriptConfigurationForTasks.create();
+      config.initScriptPattern("/jclouds-%s");
+      assertEquals(config.getBasedir(), "/");
+      assertEquals(config.getInitScriptPattern(), "/jclouds-%s");
+   }
```

Comment:
I'd also add a test to verify we now require the initscript to start with a slash. Let's add a test that verifies that the IAE is thrown.